swarm/Cargo.toml

```diff
@@ -25,6 +25,7 @@ smallvec = "1.6.1"
 void = "1"
 wasm-bindgen-futures = { version = "0.4.34", optional = true }
 getrandom = { version = "0.2.9", features = ["js"], optional = true } # Explicit dependency to be used in `wasm-bindgen` feature
+once_cell = "1.17.1"
```

Comment:
See comment in `handler.rs`. I don't think the use of `once_cell::Lazy` with `HashSet::new` is an optimization.